lib/stringify.js

```diff
@@ -20,14 +20,20 @@ var defaults = {
     delimiter: '&',
     encode: true,
     encoder: utils.encode,
+    format: 'RFC3986',
+    formatters: {
+        /** @return {string} */
```

Comment:
please remove this; i don't want jsdoc anywhere near code that i have to maintain :-)